homeassistant/components/airos/strings.json

```diff
@@ -6,12 +6,16 @@
         "data": {
           "host": "[%key:common::config_flow::data::host%]",
           "username": "[%key:common::config_flow::data::username%]",
-          "password": "[%key:common::config_flow::data::password%]"
+          "password": "[%key:common::config_flow::data::password%]",
+          "ssl": "Use https",
+          "verify_ssl": "[%key:common::config_flow::data::verify_ssl%]"
         },
         "data_description": {
           "host": "IP address or hostname of the airOS device",
           "username": "Administrator username for the airOS device, normally 'ubnt'",
-          "password": "Password configured through the UISP app or web interface"
+          "password": "Password configured through the UISP app or web interface",
+          "ssl": "Use encrypted connection",
```

Comment:
```suggestion
          "ssl": "Use encrypted connection when connecting to device. This may not be supported by devices with firmware version prior to 8.",
```

Reply:
The current implementation is for airOS v8 only (and newer when they arrive). As for the change to support non-tls; 
apparently some devices still support http (the ones I've tested with an had feedback on) only supported https. @astrandb flagged that there are still http-only devices (he has some) :)

Adding the 'when connecting to the device'

Reply:
Reworded accordingly, thanks for the sharp eye